### ISSUE TITLE
Will there be no more updates for recovered cases?

### ISSUE THREAD

**Issue by ivanMSC** [CSSEGISandData/COVID-19#1334]
From the timeseries readme:
> ---DEPRICATED WARNING---
> The files below will no longer be updated. With the release of the new data structure, we are updating our time series tables to reflect these changes. Please reference time_series_covid19_confirmed_global.csv and time_series_covid19_deaths_global.csv for the latest time series data.
> 
> time_series_19-covid-Confirmed.csv
> time_series_19-covid-Deaths.csv
> time_series_19-covid-Recovered.csv

I see no time_series_covid19_recovered_global.csv.

**Comment by daniel-karl** [comment]
Yes, what happened? We need this data!

**Comment by dpwillemin85** [comment]
I'd like to know the answer to this question as well. This is an important metric.

**Comment by jtene001** [comment]
Agreed. I was using this data to get an approximation of active cases. Without it, no clue.

**Comment by ivanMSC** [comment]
Anyone know any other source for timeseries? Time to leave this place, apparently.

**Comment by ulklc** [comment]
@ivanMSC I've started to collect time series data for country basis if you interest
https://github.com/ulklc/covid19-timeseries

**Comment by ivanMSC** [comment]
@ulklc That is awesome! You have ISO codes! I will be using this. Thank you very much.

**Comment by ivanMSC** [comment]
@ulklc there is a bug in the raw report (at least). Please see your issues page.

**Comment by laplacian-k** [comment]
What is the main reason that the number of recovered patients data is deprecated?
I don't understand yet.

**Comment by ivanMSC** [comment]
@laplacian-k https://github.com/CSSEGISandData/COVID-19/issues/1250#issuecomment-602271179

**Comment by Samsonizor** [comment]
@ivanMSC well that really sucks, but I guess I can understand why. I suppose we can close this issue now that the question is answered.

**Comment by chrisjbillington** [comment]
@ulklc your dataset is excellent, I'm switching [my plots](https://chrisbillington.net/COVID) to use it immediately. Thanks so much.
